### ISSUE TITLE
Client not provisioned for newer iOS devices

### ISSUE THREAD

**Issue by kaimbanks1** [dparnell/rethink-db-client#1]
Hi - thank you for taking the time to put together this rethinkDB client!

I've built the framework in xCode, but when I drag the RethinkDbClient.framework file to my iOS project, I get a "clang" error because the architecture has not been updated for new i386 devices. 

Not sure how to go about fixing this. Any help would be appreciated!

**Comment by dparnell** [comment]
Hi, to use it with iOS you need to use a static library as loading frameworks isn't supported for iOS apps.

**Comment by dparnell** [comment]
Hi again, I've just pushed a new version that includes a target that should work under iOS.  It compiles, but I've not tested it at all.  Let me know how it goes :smile:

**Comment by kaimbanks1** [comment]
Hi - thank you for the quick response! This functionality is key in my app, so I really appreciate it. 

The new version looks good, but I'm running into an issue where I can't run git submodule init/update because my publickey gets denied (and I can't do anything to fix it because I'm on a github enterprise account). This means that the appropriate protobuf files aren't copied into the project. 

What are the files that are supposed to be in the /Protocol directory? Is it just libProtocolBuffers.a (generated with your own protobuf-objc.git project), or are other files required? Thanks a ton.

**Comment by kaimbanks1** [comment]
I think I figured out my previous issue - I just put the /src folder in your protobuf-objc github project into the /Protocol/protobuf-objc folder, and everything compiles fine. However, when I compile the project for iOS (StaticRethinkDbClientiOS -> iOS device), no framework is generated (even though the .framework file is generated when I run it for Mac). Is there a reason for this? It may be user error - this is my first time implementing a static library. libStaticRethinkDBClient.a and RethinkDbClient.framework generate fine, but libStaticRethinkDBClientIOS.a remains red. Thank you again for your time.

**Comment by dparnell** [comment]
I just had a look and I see the same thing here.  It's weird as the static lib file is created on the disk in the correct place but the XCode UI doesn't update correctly for it.  I created a new static lib iOS project in XCode and it does the same thing.  It looks like a bug in XCode.  One thing I was able to do was change the product output dir to be the same as the OSX targets and then everything shows up as expected.  I'm not sure what will happen when trying to run this project however.  I've pushed up the new version with this change here:  ce6e7ccf3552d5e2e21e4b9ca302f0a437fb65ae
